### ISSUE TITLE
Does this project is dead?

### ISSUE THREAD

**Issue by scramatte** [charliekassel/vuejs-datepicker#821]
Hello,

Does this project is dead? 
In such case does anybody maintain another repository ?

Regards

**Comment by MrWook** [comment]
Hey, 
yes this repo is pretty much dead. Though the author has said that we can create pull request but he doesn't merge anything. 
My company proposed to take over the project but he refused https://github.com/charliekassel/vuejs-datepicker/issues/764

But we published a fork of the project with a lot of fixes and improvements https://github.com/sumcumo/vue-datepicker

**Comment by scramatte** [comment]
Hi,
Thank you to reply. I've found this one too [hokify/vuejs-datepicker](https://github.com/hokify/vuejs-datepicker)
We have published a new schema-driven form generator that use this component among other.
Take a look to [http://scramatte.github.io/eom-form-docs](http://scramatte.github.io/eom-form-docs)

I will take a look to your fork ASAP. 
Regards

**Comment by wmcmurray** [comment]
Some people have also [proposed their help](https://github.com/charliekassel/vuejs-datepicker/issues/679) as co-maintainers, but the author is apparently thinking about it since [february 2019](https://github.com/charliekassel/vuejs-datepicker/issues/679#issuecomment-463611830)... :hourglass_flowing_sand: 

I totally understand that a person don't have time to spare on his open source project, but I can't conceive why a person would prefer to hold his project back like that... making the open source ecosystem a mess.

First time ever I stumbled upon that stubbornness since 10+ years of web development. :shrug:

**Comment by mst101** [comment]
I just had a look at [Mr. Wook's fork](https://github.com/sumcumo/vue-datepicker) of this project and he appears to introduced a number of new errors e.g.[stripping out a whole of props](https://github.com/sumcumo/vue-datepicker/issues/30)... Proceed with care...

**Comment by MrWook** [comment]
Those are pretty harsh claims that you got there... as you can see on the [demo](https://sumcumo.github.io/vue-datepicker/demo/) page. All props are still available and documented. 
All breaking-changes from this repo are documented in a migration guide and there hasn't been any feature removed.

**Comment by mst101** [comment]
You're quite right Mr. Wook. I'm sorry - I was mistaken and take it all back. ;-)

I may even have a go at adding some keyboard support to your fork, if I ever get the time... Thanks

**Comment by mst101** [comment]
After my initial (misplaced) scepticism - and having checked out many other forks of this project - I'm firmly of the opinion that the @sum.cumo/vue-datepicker fork maintained by MrWook is by far the best successor to this project.

I'm pleased to have been able to contribute towards it, not least by implementing keyboard support (which was a much more complicated task than I had anticipated!).

Feel free to check it out at [https://sumcumo.github.io/vue-datepicker/](https://sumcumo.github.io/vue-datepicker/), raise any issues you may find - and if you like it, don't forget to give the project a little star. It's amazing what people will do for that tiny dose of dopamine ;-)

N.B. At the moment, it's still a Vue2 project, but plans are to first implement Typescript and then migrate to Vue3. Perhaps you can help?
